docs/how-to/index.rst

```diff
@@ -44,6 +44,7 @@ migrate services and more.
    migrate-auto-services
    remove-disk
    perform-cluster-maintenance
+   enable-fde
```

Comment:
We should specify the title we want in the index here (like in the explanation section), otherwise, the H1 on the page is picked up (and it's not very suitable in this case). I suggest naming it _Enable full disk encryption_.

Then, we'll have to change the rest to use imperatives rather than gerunds as well, for consistency.